images/kubemci/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN apt-get update && apt-get install -y \
 RUN cd /${GOPATH}/src && \
     mkdir -p github.com/GoogleCloudPlatform && \
     cd github.com/GoogleCloudPlatform && \
-    git clone https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress.git && \
+    git clone --depth 1 https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress.git && \
```

Comment:
For anyone else who happens to stumble upon this: shallow clones are a bad idea if you intend to keep the clone up to date over time.  It's fine in this case since the clone is removed after building its contents.